### ISSUE TITLE
Oracle x Cupid

### ISSUE THREAD

**Issue by McTsts** [McTsts/Werewolves-Roles#244]
Oracle sees the roles alignment which for cupid and lover is different than the actual alignment. Is this intentional?

**Comment by Shepherdwhen** [comment]
They should both be a solo or town wincon therefore should be seen the same. I doubt it's intentional more of an issue caused by Cupid's original alignment being UA/town

**Comment by McTsts** [comment]
I think it should probably be one of those two:
- Oracle still sees the roles alignment but is affected by strong disguises
- Oracle still is not affected by disguises but sees the actual alignment

Both of these would be more consistent with how the disguises work.

![image](https://user-images.githubusercontent.com/24660095/117663158-a1a8a580-b1a0-11eb-95bf-217c99efe897.png)

**Comment by Shepherdwhen** [comment]
I thought reporter sees how many of a specific role is alive?

**Comment by McTsts** [comment]
> I thought reporter sees how many of a specific role is alive?

That's Receptionist. That one isn't in that image because it checks a role, not a player/a player's role

**Comment by Shepherdwhen** [comment]
> > I thought reporter sees how many of a specific role is alive?
> 
> That's Receptionist. That one isn't in that image because it checks a role, not a player/a player's role

Yes I am dumb lol, I didn't see receptionist so I assumed the last one was it because I saw the rep part :s
